utils/url.ts

```diff
@@ -1,4 +1,27 @@
+import base64 from 'base-64';
```

Comment:
You may not need it https://stackoverflow.com/questions/30106476/using-javascripts-atob-to-decode-base64-doesnt-properly-decode-utf-8-strings